src/antio/libeep/__init__.py

```diff
@@ -176,6 +177,22 @@ def _get_start_time(self):
         """
         return pyeep.get_start_time(self._handle)
 
+    def _get_start_time(self):
+        """Get start time in UNIX format.
+
+        Returns
+        -------
+        start_time : float
+            Acquisition start time.
+        """
+        start_date, start_fraction = pyeep.get_start_date_and_fraction(self._handle)
+        # start date is in EXCEL format
+        if start_date >= 27538 and start_date <= 2958464:
+            start_date = np.round(start_date * 3600.0 * 24.0) - 2209161600
+        else:
+            start_date = 0
+        return start_date + start_fraction
```

Comment:
This is now overwriting the method `_get_start_time` define above, and that only adds the missing `ms`, `us` and maybe `ns`? I don't think we need this degree of precision on the measurement date, am I missing something?

Reply:
For most of the use cases, we don't. But, there could be a some cases where the event triggers cannot be recorded by the ANT eego software, but recorded separately (as in a .txt or .csv files) with timestamps. Aligning such event triggers will need the millisecond precision of start time. 

* Brainvision reader do read upto millisecond precision.

Reply:
OK, I'll push one more commit and mark for merge!